qa/suites/fs/upgrade/nofs/README

```diff
@@ -0,0 +1,3 @@
+This test just verifies that upgrades work with no file system present. In
```

Comment:
@batrick are you able to reproduce https://tracker.ceph.com/issues/51673 using this?

Reply:
No, see a405e44aef5a495c2875e03333d265dd81a05a44 commit message.